src/components/carousel/index.tsx

```diff
@@ -16,6 +20,9 @@ type SharedProps = {
   startIndex?: number;
   onSlideClick?: (index: number) => void;
   onSlideSelect?: (index: number) => void;
+  slidesPerView?: ResponsiveObject<number>;
+  loop?: boolean;
+  slidesToScroll?: 'auto' | number;
```

Comment:
It seems that slidesToScroll: auto and slidesToScroll: 1 work the same. Do we have a specific use case where these two works differently?

Reply:
no no they don't work the same. If you have multiple slides per view, it adjusts based on it.